.github/workflows/pr-checks.yml

```diff
@@ -42,3 +42,41 @@ jobs:
       - name: rust:build-browser
         run: |
           npm run rust:build-browser
+      - name: all:test
```

Comment:
probably it makes sense to have multiple checks, e.g. why to test old cml when only new crates are changed? you can specify the path for particular checks